nodeadm/test/e2e/cases/node-taints/run.sh

```diff
@@ -18,4 +18,8 @@ assert::file-contains /etc/eks/kubelet/environment "--register-with-taints=foo=:
 
 mock::kubelet 1.23.0
 nodeadm init --skip run --config-source file://config.yaml
-assert::json-files-equal /etc/kubernetes/kubelet/config.json expected-kubelet-config.json
+assert::file-contains /etc/eks/kubelet/environment "--register-with-taints=foo=:NoSchedule,foo2=baz:NoSchedule"
+
+mock::kubelet 1.28.0
+nodeadm init --skip run --config-source file://config.yaml
```

Comment:
Would be sweet if we could do 
```
cat <<EOF | kubectl init -f -
... inline file
EOF
```

Reply:
most users wont have a need to actually call `nodeadm` based on how its booted in the ami, but im a fan of the style